### PR TITLE
Simple fix Gradle run problem

### PR DESCRIPTION
**Pull request by BenCheung0422**
Mentioned by: https://github.com/MinicraftPlus/minicraft-plus-revived/pull/403#issuecomment-1363594648

### CODE REVIEW

**Comment by Makkkkus** [comment]
Haven't we already made a file handler for this exact problem?

**Comment by BenCheung0422** [comment]
I am not sure where you meant?

**Review by ChristofferHolmesland — APPROVED** [review]

**Review by ChristofferHolmesland — DISMISSED** [review]

**Comment by KalmeMarq** [comment]
There's actually another way of fixing this and also being able to remove the listResources methods with IDE and outside and other shenanigans.
Create a dummy file in the src/main/resources/, for example ".root". Get its URI and then try to get its Path and if it fails do FileSystems.newFileSystem(rootURI, Collections.emptyMap()) and then try getting the Path again. After you got the Path of the .root file get its parent Path. Now you can use the src/main/resources folder as a file system and get files, paths, etc. Works in both IDE and in production. Nicest solution.

**Review by TheBigEye — APPROVED** [review]

**Review by ChristofferHolmesland — APPROVED** [review]

File: src/main/java/minicraft/core/io/FileHandler.java [hunk 1]
```diff
@@ -159,38 +165,28 @@ public FileVisitResult visitFileFailed(Path p, IOException ex) {
...
 
+		ArrayList<String> names = new ArrayList<>();
+		try (Stream<Path> paths = Files.walk(path)) {
+			Path finalPath = path;
+			paths.forEach(p -> names.add(finalPath.getParent().relativize(p).toString()));
 			return names;
 		} catch (IOException e) {
-			CrashHandler.errorHandle(e);
-			return names;
+			throw new RuntimeException(e); // FATAL ERROR
```
Comment (Makkkkus) [inline src/main/java/minicraft/core/io/FileHandler.java:189]: A fatal error should probably stop the game. That's what fatal means.

**Review by Makkkkus — COMMENTED** [review]
Looks good, but I can't approve yet.

File: src/main/java/minicraft/core/io/FileHandler.java (hunk 1 above)
Comment (BenCheung0422) [reply src/main/java/minicraft/core/io/FileHandler.java:189]: Currently, there is no crash report system, only popups are used. This error is thrown, unhandled error is caught by the handler.

**Review by ChristofferHolmesland — APPROVED** [review]

**Pull request merged**